[2024-11-02T21:28:43.441+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-11-02T21:28:43.498+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-11-02T21:28:43.499+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-11-02T21:28:49.466+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotSchemaSubmitOperator): submit_schemas> on 2024-09-20 00:00:00+00:00
[2024-11-02T21:28:55.041+0000] {standard_task_runner.py:57} INFO - Started process 12594 to run task
[2024-11-02T21:28:56.356+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'schema_dag', 'submit_schemas', 'scheduled__2024-09-20T00:00:00+00:00', '--job-id', '199', '--raw', '--subdir', 'DAGS_FOLDER/schema_dag.py', '--cfg-path', '/tmp/tmp6_vocthm']
[2024-11-02T21:28:56.642+0000] {standard_task_runner.py:85} INFO - Job 199: Subtask submit_schemas
[2024-11-02T21:29:02.227+0000] {task_command.py:415} INFO - Running <TaskInstance: schema_dag.submit_schemas scheduled__2024-09-20T00:00:00+00:00 [running]> on host f79a3875bd34
[2024-11-02T21:29:05.700+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ctwang' AIRFLOW_CTX_DAG_ID='schema_dag' AIRFLOW_CTX_TASK_ID='submit_schemas' AIRFLOW_CTX_EXECUTION_DATE='2024-09-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-20T00:00:00+00:00'
[2024-11-02T21:29:06.202+0000] {pinot_schema_operator.py:35} ERROR - An error occurred: HTTPConnectionPool(host='pinot-controller', port=9000): Max retries exceeded with url: /schemas (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f21fcd6e0d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2024-11-02T21:29:07.135+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=schema_dag, task_id=submit_schemas, execution_date=20240920T000000, start_date=20241102T212843, end_date=20241102T212907
[2024-11-02T21:29:07.788+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-02T21:29:10.615+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
